### PR TITLE
[ADD] hide the purchase requisition generic groups configuration at the res.user form view to only use this module requisition security groups

### PR DESCRIPTION
**Pull request by zaoral**
Task [vx#3351](https://www.vauxoo.com/#id=3351&view_type=form&model=project.task) HU [319](https://www.vauxoo.com/?debug=#id=319&view_type=form&model=user.story&action=726) Sprint 10
Result at this [image](https://docs.google.com/a/vauxoo.com/file/d/0B1UUUA6bRx8-VFRoblkwVXVubG8/edit?usp=drivesdk)

### CODE REVIEW

**Comment by nhomar** [comment]
Esto está incorrectamente.

El nombre del grupo que usas en el XML no existe, debes usar el xml real.

File: purchase_requisition_for_everybody/view/purchase_requisition_view.xml [hunk 1]
```diff
@@ -59,5 +59,16 @@
         action="purchase_requisition.action_purchase_requisition"
         />
 
+    <record model="ir.ui.view" id="res_users_hide_requistion_group">
+        <field name="name">res.users.hide.requistion.group</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.user_groups_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='sel_groups_54_55']" position="attributes">
```
Comment (nhomar) [inline purchase_requisition_for_everybody/view/purchase_requisition_view.xml:67]: Anda a usuarios > grupos, busca el grupo y asigna aquí en xml_id correcto.

Comment (zaoral) [inline purchase_requisition_for_everybody/view/purchase_requisition_view.xml:67]: @nhomar in the xml of the original view I found [this](https://github.com/vauxoo/odoo/blob/8.0/openerp/addons/mabase/res/res_users_view.xml#L238-L245) and I check for another view that inherit from the principal but I dont found any.

I search for the xml id of the group as you say but when I added the next error was thow

```
Error details:
Element '<xpath expr="//field[@name='purchase_requisition.group_purchase_requisition_manager']">' cannot be located in parent view
```

I dont now how to manage this.Can you help me please?
Regards

Comment (zaoral) [inline purchase_requisition_for_everybody/view/purchase_requisition_view.xml:67]: here the patch that fails.

```
diff --git a/purchase_requisition_for_everybody/view/purchase_requisition_view.xml b/purchase_requisition_for_everybody/view/purchase_requisition_view.xml
index 3ab32bf..3ce7539 100644
--- a/purchase_requisition_for_everybody/view/purchase_requisition_view.xml
+++ b/purchase_requisition_for_everybody/view/purchase_requisition_view.xml
@@ -64,7 +64,10 @@
         <field name="model">res.users</field>
         <field name="inherit_id" ref="base.user_groups_view"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='sel_groups_54_55']" position="attributes">
+            <xpath expr="//field[@name='purchase_requisition.group_purchase_requisition_manager']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//field[@name='purchase_requisition.group_purchase_requisition_user']" position="attributes">
                 <attribute name="invisible">True</attribute>
             </xpath>
         </field>
```

**Comment by nhomar** [comment]
Hello.

Due to this is a low important behavior, we left this PR open, but blocked until we find a correct approach to "hide a group" the way @zaoral is doing thing here works technically but conceptually I think is wrong.

That's why you see this PR marked as blocked, one time we find a better approach we unblock it and apply it.

**Comment by luisg123v** [comment]
Has conflicts and abandoned, closing.

**Pull request closed without merging**